.github/workflows/test-install.yml

```diff
@@ -90,15 +90,17 @@ jobs:
     - run: python --version
     - run: which python
 
-    - name: Create conda environment
+    - name: Test direct installation
       run: |
-        conda env create -f environment.yml -n test-environment
-        source activate test-environment
-        python -m pip install --no-deps -e .
+        conda env create -n test-install aiida-core
```

Comment:
Is that supposed to install aiida-core into the new environment?

When I run it locally, I get
```
$ conda env create -n test-install  aiida-core

SpecNotFound: Invalid name, try the format: user/package
```

Also, I guess it's good to test this but if there are issues with the conda-forge feedstock for aiida-core they will need to be fixed in the respective repository (cannot be fixed here).

Reply:
Hmm.. odd, it appears to have worked even with the `env` command on the runner, however I updated it to use the more standard (?) `create` command.